cli/src/core/parse/profile.go

```diff
@@ -0,0 +1,48 @@
+package parse
+
+import (
+	"cli/core"
+	"cli/core/state"
+
+	"github.com/luno/jettison/errors"
+	"github.com/spf13/cobra"
+)
+
+func GetPackageSpecFromProfile(cmd *cobra.Command, config core.Config, packageSpec core.PackageSpec) (*core.PackageSpec, error) {
+	profile := core.Profile{}
+	profileName, err := cmd.Flags().GetString("profile")
+	if err != nil {
+		return nil, errors.Wrap(err, "")
+	}
+
+	for _, p := range config.Profiles {
+		if p.Name == profileName {
+			profile = p
+			break
+		}
+	}
```

Comment:
Think we should return an error when the profile name given is invalid

Reply:
That is done in the validate function introduced in https://github.com/openhie/package-starter-kit/pull/36